### ISSUE TITLE
Further exploration on In-App Notification system

### ISSUE THREAD

**Issue by nanyangpro** [bcgov/cas-cif#1008]
#### Describe the task
Associated Design for Amendments/Revisions: https://www.figma.com/file/PzOY8RyQnVXnjZJdSv2AqQ/Wireframing_Figma?node-id=2765%3A3314

The CIF users want to alert other user groups on certain action items in their process. #932 addressed some aspects below, the remaining require further investigation to design visual components and workflows for such Notification system.
- **In-App** vs **Email** (#932)
- **Amendment-Specific** (#932) vs **Project-Level (or even Program-Level)**
- **User Management** (Ops Team / Tech Team / **Director** / Proponent)

#### Acceptance Criteria
- [ ] Investigate user needs and expectations from the CIF team
- [ ] Create static mock-ups for further discussion/validation
- [ ] Create interactive prototypes to visualize the envisioned workflows
- [ ] Validate and finalize design with CIF and the devs
- [ ] Write up associated user story tickets with design/prototype links for dev refinement and implementation

#### Additional context

- Add any other context about the task here.
- Or here

**Comment by LindsayMacfarlane** [comment]
Hi @nanyangpro - I'm wondering if for next sprint, we can pull this ticket back into the design backlog? We have a lot on our plate and I want to be mindful of your time.

**Comment by nanyangpro** [comment]
Good call, @LindsayMacfarlane - Yes. Considering the priority of it and tickets #1137 and #1139 for the next sprint, let's move it back to the design backlog. Thanks!

**Comment by nanyangpro** [comment]
Closed as outdated.
